docs/manual/guides/deployer.en.md

```diff
@@ -19,7 +19,7 @@ If not done yet, install Deployer as described here: [https://deployer.org/docs/
 
 You can either install Deployer globally or per project and use the command `dep` or `./vendor/bin/dep` respectively.
```

Comment:
The global install is not recommended anymore. Instead, it is recommended to install Deployer per project and create a bash alias (this changed in rc.6 or so).

See https://deployer.org/docs/7.x/installation

Can you updated this as well?

Reply:
Done in [93eaf6f](https://github.com/contao/docs/pull/1071/commits/93eaf6f232b1063cbf0e07b6166fed9f889f8df5)